### PR TITLE
Better cities

### PR DESCRIPTION
**Pull request by kent1D**
Better differentiation between different kind of cities at bigger zoom

Better datas and position of cities issued of our datas

When a city doesn't have a name, don't show its point (visible in Saudi Arabia for example, at the north west of Riyad)

### CODE REVIEW

File: labels.mss [hunk 1]
```diff
@@ -71,11 +70,17 @@
...
     shield-fill: @city_text;
     shield-face-name: @bold;
-    shield-size: 12;
+    shield-size: 13;
   }
   [type='embassy'] {
     shield-allow-overlap: true;
   }
+  [type='intermediate'] {
+    shield-face-name: @medium;
```
Comment (yohanboniface) [inline labels.mss:79]: Humm, why aren't those statements at the root?

**Comment by yohanboniface** [comment]
One thing that can help me, as a reviewer: when doing CartoCSS changes, some before/after pictures could help me to understand the intention better :)
This is easy to do with kosmtik-compare and using the prod URL as the compare map.

File: forte.yml [hunk 2]
```diff
@@ -310,7 +310,9 @@ Layer:
           FROM
             osm_places
           WHERE
-            type IN ('state', 'city', 'town') AND geometry && !bbox!
+            name != ''
```
Comment (kent1D) [inline forte.yml:313]: capture d ecran 2018-02-21 a 15 20 35

Here is one example of a point missing "name" (between Buraydah and Qasim)

Comment (yohanboniface) [reply forte.yml:313]: That one was not that hard to understand ;)
My remark was more about when dealing with font-size and such :)

Comment (kent1D) [reply forte.yml:313]: I know, I just tested the functionality of kosmtik, I need to reimport a better country to make the screenshots but was lazy

Comment (yohanboniface) [reply forte.yml:313]: :)

**Review by kent1D — COMMENTED** [review]
Don't know if it was done on purpose, but I really think it's missing

capture d ecran 2018-02-23 a 08 54 47

Ghadamis for example here

File: piano.yml [hunk 3]
```diff
@@ -307,7 +307,7 @@ Layer:
             AND type IN ('state', 'city', 'town')
             AND geometry && !bbox!
           ORDER BY
-            prio
+            prio, population DESC
```
Comment (kent1D) [inline piano.yml]: capture d ecran 2018-02-23 a 09 50 27

Valencia in Spain is much more bigger than Silla

File: labels.mss (hunk 1 above)
Comment (kent1D) [reply labels.mss:79]: By default, town, it's regular

Intermediate, is at the moment a small subset of cities, isn't it

**Comment by kent1D** [comment]
Here is a screenshot of the main graphical change (have a look at Barcelone)

capture d ecran 2018-02-23 a 13 31 37

**Pull request merged**